### PR TITLE
Improvments and translation

### LINKED ISSUE #3: Appstream tests fail
AppCenter ran a bunch of tests on your appstream file. These are the results.

## Warnings:
Most of these are fixed during the building process, but you should ensure they
are fixed in your code for future releases.
  - Missing "project_license" field

### For more information, see:
- [The AppStream Quickstart Guide](https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html)
- [The AppStream Metadata Specification](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html)

### Build Information
Affects Architecture: amd64
Affects Distribution: juno
Built with the following references:
- refs/tags/0.1.2

### PR DESCRIPTION
**Pull request by NathanBnm**
Hi! I made some changes to your app to make it more compliant with the elementary guidelines.
I updated the way the meson build was organized and added ability to translate your app.

By the way, as a French student, I added French translation too.

Let me know if you have any questions :smile:

**This PR is ready for review**

[EDIT: I added the missing license in appstream so this PR will resolve #3]

### CODE REVIEW

**Comment by ryonakano** [comment]
For some reason, both `appdata.xml` and `appdata.xml.in` seem to be added.

File: meson/post_install.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+import os
+import subprocess
+
+schema_dir = os.path.join(os.environ['MESON_INSTALL_PREFIX'], 'share', 'glib-2.0', 'schemas')
+icon_cache_dir = os.path.join(os.environ['MESON_INSTALL_PREFIX'], 'share', 'icons', 'hicolor')
+
+if not os.environ.get('DESTDIR'):
+    print('Updating desktop icon cache…')
+    subprocess.call(['gtk-update-icon-cache', '-qtf', icon_cache_dir])
```
Comment (RyanAfrish7) [inline meson/post_install.py:11]: Interesting fix to get the icons updated.

**Comment by NathanBnm** [comment]
Thanks for the notice @ryonakano I removed it

**Review by RyanAfrish7 — APPROVED** [review]
Thanks for the PR. Never did I18N for linux apps before. Just curious on how to maintain this, especially the [code references](https://github.com/RyanAfrish7/satellite/pull/4/files#diff-07023e7a98cd6472148f7114d7d2df01R24) on the .pot files.

**Comment by ryonakano** [comment]
> Just curious on how to maintain this, especially the code references on the .pot files.

@RyanAfrish7 If you update .pot files these references will be automatically updated. To update the .pot file, run:

```
cd build/
ninja <The project name defined in meson.build>-pot
```

And run the following command when you want to update all translation files including the .pot file and po files:

```
cd build/
ninja <The project name defined in meson.build>-update-po
```

Note: If you want to update the translation files of the app itself, then `<The project name defined in meson.build>` should be replaced by `com.github.ryanafrish7.satellite` (defined [here](https://github.com/NathanBnm/satellite/blob/c00509de3c292c79863d12a32051cc82768af766/po/meson.build#L1)), so the command looks like this:

```
ninja com.github.ryanafrish7.satellite-update-po
```

And if you update the metadata files ones (strings in .appdata.xml and .desktop files), the following command will work (defined [here](https://github.com/NathanBnm/satellite/blob/c00509de3c292c79863d12a32051cc82768af766/po/extra/meson.build#L1)):

```
ninja extra-update-po
```

See also: https://elementary.io/docs/code/getting-started#Adding-Translations

**Comment by ryonakano** [comment]
@NathanBnm BTW, your fix for #3 seems to be gone by that removal…😥

**Comment by NathanBnm** [comment]
@ryonakano oh okay I didn't notice, I'm going to check, I'll do a new PR

**Comment by ryonakano** [comment]
@NathanBnm It seems @RyanAfrish7 has just fixed it in ef36138d7113988db0495a0e1f4d565949520ff6

**Comment by NathanBnm** [comment]
Great!

**Pull request merged**